cert/cert.go

```diff
@@ -443,6 +479,34 @@ func (spec *Spec) Certificate() *x509.Certificate {
 	return spec.tr.Provider.Certificate()
 }
 
+// CertificateAge returns age of certificate associated with spec
+func (spec *Spec) CertificateAge() time.Duration {
+	c := spec.Certificate()
+	if c == nil {
+		return -1
+	}
+	now := time.Now()
+	return now.Sub(c.NotBefore)
+}
+
+// CA age returns age of CA associated with Spec
+func (spec *Spec) CAAge() time.Duration {
+	c := spec.CA.GetPEM()
```

Comment:
When it's defined you say "GetPEM is for testing only!" but `CAAge` is used by not-test code - should you be using a different function here or does the comment just need to be clarified?

Reply:
Good catch! Was initially for testing but ended up needing for another function -- will update the comment